types/dragino/lht65n/uplink.js

```diff
@@ -93,7 +93,7 @@ function decoder(bytes, port) {
   const pollMessageStatus = (bytes[6] >> 7) & 0x01;
   const connect = (bytes[6] & 0x80) >> 7;
   const decode = { lifecycle: {}, decoded: {}, external: {}, datalog: {} };
-  if (((port === 3) & (bytes[2] === 0x01)) | (bytes[2] === 0x02)) {
+  if (((port === 3) & ((bytes[2] === 0x01) | (bytes[2] === 0x02))) {
```

Comment:
possibly align this with the logic provided in the Dragino repository? 
https://github.com/dragino/dragino-end-node-decoder/blob/main/LHT65N/LHT65N%20V1.5.0%20Chirpstack%20%204.0%20decoder.txt#L101

Reply:
Out of my view yes. But I could find only the (bytes[2] === 0x01) in the official dragino documentation in the alarmin section. Don't know what the others are.

Important is, that (port === 3) is always in an AND condition with the additional bytes. But with the provided dragino code from chirpstack, that's the case.